### ISSUE TITLE
Migrate to Elm v0.19

### ISSUE THREAD

**Issue by alexanderkiel** [NoRedInk/list-selection#9]
I migrated the code and package description to Elm v0.19 in this [commit][1]. I also published the package under my name [here][2]. If I understand things correctly, this package has to be published under a new name anyway because one has to start with version 1.0.0 and the tags on GitHub have to match. So I can't issue a pull request. I think you have to create a new repo for the 0.19 version.

[1]: <https://github.com/alexanderkiel/list-selection/commit/272d1d555d6c146af50b00efc4131b2910d39bbd>
[2]: <https://package.elm-lang.org/packages/alexanderkiel/list-selection/latest/>

**Comment by pzp1997** [comment]
Unless I'm mistaken, you don't need to start with version 1.0.0 when updating a package for 0.19. The reason why `core` and other packages reset to 1.0.0 is because the owner changed from elm-lang to elm.

**Comment by alexanderkiel** [comment]
@pzp1997 Yes you are right. I had to start with 1.0.0 because I tried to publish the package under my own name.
